doc/releases/changelog-0.41.0.md

```diff
@@ -220,323 +240,494 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
-  >>> U_sparse = sp.sparse.csr_matrix(U_dense)
-  >>> op = qml.QubitUnitary(U_sparse, wires=[0, 1])
-  >>> print(op.sparse_matrix())
-  <Compressed Sparse Row sparse matrix of dtype 'float64'
-          with 4 stored elements and shape (4, 4)>
-    Coords        Values
-    (0, 0)        1.0
-    (1, 1)        1.0
-    (2, 2)        1.0
-    (3, 3)        1.0
-  >>> op.sparse_matrix().toarray()
-  array([[1., 0., 0., 0.],
-        [0., 1., 0., 0.],
-        [0., 0., 1., 0.],
-        [0., 0., 0., 1.]])
+  >>> f(0.1, 0.2)
+  I constructed plxpr
+  0.97517043
+  >>> f(0.1, 0.3)
+  0.9505638
+  >>> f(0.2, 0.3)
+  I constructed plxpr
+  0.93629336
   ```
 
-* `qml.StatePrep` now accepts sparse state vectors. Users can create `StatePrep` using `scipy.sparse.csr_matrix`. Note that non-zero `pad_with` is forbidden.
-  [(#6863)](https://github.com/PennyLaneAI/pennylane/pull/6863)
+<h4>End-to-end Sparse Execution 🌌</h4>
+
+* Sparse data structures in compressed-sparse-row (csr) format are now supported end-to-end
+  in PennyLane, resulting in faster execution for large sparse objects.
+  [(#6883)](https://github.com/PennyLaneAI/pennylane/pull/6883) 
+  [(#7139)](https://github.com/PennyLaneAI/pennylane/pull/7139) 
+  [(#7191)](https://github.com/PennyLaneAI/pennylane/pull/7191)
+
+  Sparse-array input and execution are now supported on `default.qubit` and `lightning.qubit`
+  with a variety of templates, preserving sparsity throughout the entire simulation.
+  
+  Specifically, the following templates now support sparse data structures:
+
+  * :class:`qml.StatePrep <pennylane.StatePrep>`
+    [(#6863)](https://github.com/PennyLaneAI/pennylane/pull/6863)
+  * :class:`qml.QubitUnitary <pennylane.QubitUnitary>` 
+    [(#6889)](https://github.com/PennyLaneAI/pennylane/pull/6889)
+    [(#6986)](https://github.com/PennyLaneAI/pennylane/pull/6986)
+    [(#7143)](https://github.com/PennyLaneAI/pennylane/pull/7143)
+  * :class:`qml.BlockEncode <pennylane.BlockEncode>`
+    [(#6963)](https://github.com/PennyLaneAI/pennylane/pull/6963)
+    [(#7140)](https://github.com/PennyLaneAI/pennylane/pull/7140)
+  * :class:`qml.SWAP <pennylane.SWAP>`
+    [(#6965)](https://github.com/PennyLaneAI/pennylane/pull/6965)
+  * :func:`Controlled <pennylane.ctrl>` operations
+    [(#6994)](https://github.com/PennyLaneAI/pennylane/pull/6994)
+  
+  ```python
+  import scipy
+  import numpy as np
+
+  sparse_state = scipy.sparse.csr_array([0, 1, 0, 0])
+  mat = np.kron(np.identity(2**12), qml.X.compute_matrix())
+  sparse_mat = scipy.sparse.csr_array(mat)
+  sparse_x = scipy.sparse.csr_array(qml.X.compute_matrix())
+
+  dev = qml.device("default.qubit")
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.StatePrep(sparse_state, wires=range(2))
+
+      for i in range(10):
+          qml.H(i)
+          qml.CNOT(wires=[i, i + 1])
+
+      qml.QubitUnitary(sparse_mat, wires=range(13))
+      qml.ctrl(qml.QubitUnitary(sparse_x, wires=0), control=1)
+      return qml.state()
+    ```
```

Comment:
````suggestion
    ```python
    import scipy
    import numpy as np

    sparse_state = scipy.sparse.csr_array([0, 1, 0, 0])
    mat = np.kron(np.identity(2**12), qml.X.compute_matrix())
    sparse_mat = scipy.sparse.csr_array(mat)
    sparse_x = scipy.sparse.csr_array(qml.X.compute_matrix())

    dev = qml.device("default.qubit")

    @qml.qnode(dev)
    def circuit():
        qml.StatePrep(sparse_state, wires=range(2))

        for i in range(10):
            qml.H(i)
            qml.CNOT(wires=[i, i + 1])

        qml.QubitUnitary(sparse_mat, wires=range(13))
        qml.ctrl(qml.QubitUnitary(sparse_x, wires=0), control=1)
        return qml.state()
    ```
````

Reply:
Is this to make it in-line with the individual op bullet points?
Conceptually I think this as fitting under the previous bullet point and so less indented :thinking: